src/index.js

```diff
@@ -81,6 +81,7 @@ function App() {
     // It'll automatically check if userToken is available and push back to login if not :)
     history.push('/login');
   };
+  // eslint-disable-next-line
   const [corsErrorModalOpen, setCorsErrorModalOpen] = React.useState(false);
```

Comment:
One way to avoid random 'eslint-disable-next-line' comments is to also add a comment on _why_ corsErrorModalOpen is not yet being used.

Reply:
Good advice - comments have been added to every //eslint ignore line